website/src/css/global.css

```diff
@@ -167,6 +168,10 @@ hr.footnotes-sep {
   box-shadow: inset 3px 0 0 0 rgb(201, 167, 255);
 }
 
+table {
+  display: block;
+}
+
```

Comment:
why this? it breaks the styling for tables.

Reply:
`overflow-x: auto;` only applies to block elements, so without this the `.content` will overflow instead of just the table.

Can I get a link/screenshot of that breakage as I'm not seeing it?

<img width="658" alt="Example screenshot of a table from the documentation." src="https://user-images.githubusercontent.com/1073681/59204744-a21b1900-8be4-11e9-99f4-e9f8bfb1e3ab.png">

Reply:
https://deploy-preview-251--saber.netlify.com/docs/markdown-features.html#tables

<img width="702" alt="屏幕快照 2019-06-10 下午11 07 05" src="https://user-images.githubusercontent.com/8784712/59205018-7c861380-8bd4-11e9-8b00-9c6b6a1f0db8.png">